helmfile.d/snippets/defaults.yaml

```diff
@@ -143,24 +150,113 @@ environments:
           prometheus-blackbox-exporter: {}
           prometheus:
             enabled: false
+            remoteWrite:
+                enabled: false
+            otomiThanos: false
             replicas: 1
+            role: standalone
             scrapeInterval: 60s
             retentionSize: 4GB
             storageSize: 5Gi
           redis-shared:
             enabled: false
+          thanos:
+            enabled: false
+            bucketweb:
+              resources:
+                requests:
+                  cpu: 100m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+            query:
+              replicaCount: 1
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: '1'
+                  memory: 1Gi
+            queryFrontend:
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 512Mi
+            receiver:
+              mode: standalone
+              replicaCount: 1
+              persistence:
+                size: 8Gi
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: '1'
+                  memory: 1Gi
+            storegateway:
+              persistence:
+                size: 8Gi
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+            compactor:
+              resources:
+                requests:
+                  cpu: 200m
+                  memory: 128Mi
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+              retentionResolutionRaw: 30d
```

Comment:
> As a rule of thumb retention for each downsampling level should be the same, and should be greater than the maximum date range (10 days for 5m to 1h downsampling).
https://thanos.io/tip/components/compact.md/#-downsampling-note-about-resolution-and-retention-

Could you explain why do you chose `retentionResolution1h: 10y`